cow_amm/cow_amm_lp_and_tvl_4047078.sql

```diff
@@ -0,0 +1,151 @@
+-- Computes the TVL, lp token total supply and lp token price of a CoW AMM pool over time
+-- Parameters
+--  {{token_a}} - either token of the desired uni pool
+--  {{token_b}} - other token of the desired uni pool
+--  {{start}} - date as of which the analysis should run
+
+-- Given that we might not have records every day in the source data (e.g. not every day the lp supply may change), 
+-- but still want to visualize development on a per day basis,  we create an auxiliary table with one record per 
+-- day between `start` and `now`
+with date_range as (
+    select t.day
+    from
+        unnest(sequence(
+            date(timestamp '{{start}}'),
+            date(now())
+        )) t (day) --noqa: AL01
+),
+
+-- Finds the CoW AMM pool address given tokens specified in query parameters (regardless of order) 
+cow_amm_pool as (
+    select
+        blockchain,
+        address,
+        token_1_address,
+        token_2_address
+    from query_3959044
+    where ((token_1_address = {{token_a}} and token_2_address = {{token_b}}) or (token_2_address = {{token_a}} and token_1_address = {{token_b}}))
+),
+
+-- per day lp token total supply changes of the CoW AMM pool by looking at burn/mint events
+lp_balance_delta as (
+    select
+        date(evt_block_time) as "day",
+        sum(case when "from" = 0x0000000000000000000000000000000000000000 then value else -value end) as lp_supply
+    from erc20_ethereum.evt_transfer
```

Comment:
Yes, I asked Olga to focus on mainnet so we have some data and can make sure we are presenting the right thing, but this can be changed to

```
erc20_{{chain}}.evt_transfer
```

and we can make all performance queries chain agnostic (should be a separate PR imo)